add existing protocol sfp-406-nl

ToonHub opened the pull request:
<!--- indicate the protocol code in the Title above -->

## Description

This is an existing protocol that I converted from word-format.

Below you can download the rendered version. Unzip the file and open index.html or the pdf-file.
[sfp_406_nl_vegopname_luronium.zip](https://github.com/inbo/protocolsource/files/11552377/sfp_406_nl_vegopname_luronium.zip)

I get an error with `check_frontmatter()`: 

> version` number in the YAML of index.Rmd needs to be updated. 

However version number seems OK to me (when I update the version number, I get the same error). 

<!--- Describe your protocol proposal or your proposed changes-->

<!--- You can mention collaborators with "@githubname"-->

## Related Issue

<!--- if this closes an issue make sure include e.g., "closes #4" or similar - or if just relates to an issue make sure to mention it like "#4" -->

## Task list

<!--see https://docs.github.com/en/github/managing-your-work-on-github/about-task-lists for an explanation on how to use task lists-->

Steps by contributor:

-   [x] Add description to this pull request (under "## Description")
-   [ ] Open the dropdown (triangle) near 'create pull request' and choose 'Create draft pull request' **--> I can't find this** 
-   [x] Check for potential problems by running `protocolhelper::check_frontmatter()` and address them
-   [x] Check for potential problems by running `protocolhelper::check_structure()` and address them
-   [x] Add further commits if needed and push them to GitHub
-   [x] Update the protocol-specific `NEWS.Rmd`
-   [x] Mark the pull request as 'ready for review'

Review steps for the author(s):

-   [x] Add reviewers, at least one subject-matter specialist and one administrator 
-   [x] Wait for review comments and address them
-   [ ] Iterate until reviewer approvals (merging the pull request will be done by an administrator)
-   [ ] Verify that the checks done by continuous integration succeeded. These will check if `protocolhelper::check_frontmatter()` and `protocolhelper::check_structure()` succeeded without errors.

To be done by an administrator after review: see [guidelines for admins](https://github.com/inbo/protocolsource/blob/main/RELEASES.md).

anleyssen commented:
I reviewed “index.html” and screened the pdf-version. Some comments:

- link of ‘deze tabel’ in paragraph ‘metadata’ doesn’t seem to work

- is it possible to add clickable links for the other protocols (e.g. sfp-113); probably work for the future, as these are not yet available on the github-page

- the link of ‘bijlage 2’ is not correct; it does not direct to the list of locations 

Other parts seem to be ok.

ToonHub commented:
I corrected the link in bijlage 2

Protocol sfp-113-nl (secchi diepte) is now available on the protocol website. I added a link to this protocol in spf-406-nl

@anleyssen @hansvancalster: thanks for reviewing the changes and approving the pull request when OK
 
[sfp_406_nl.zip](https://github.com/inbo/protocolsource/files/11815332/sfp_406_nl.zip)

hansvancalster commented on source/sfp/4_vegetation/sfp_406_nl_vegopname_luronium/05_competenties.Rmd:
````diff
@@ -0,0 +1,31 @@
+```
+De veldwerker is vertrouwd met volgende deelprotocollen:
+
+-   veiligheid in en rond water (sfp-112; protocol te ontwikkelen);
````
referentie naar sfp-112 behouden of verwijderen of ...

hansvancalster commented on source/sfp/4_vegetation/sfp_406_nl_vegopname_luronium/05_competenties.Rmd:
```diff
@@ -0,0 +1,31 @@
+
+-   bepaling doorzicht waterkolom oppervlaktewater op basis van de Secchi-diepte ([sfp-113 2023.04](../2023.04/index.html));
+
+-   bioveiligheidsmaatregelen (sfp-015, protocol te ontwikkelen).
```
zelfde opmerking

hansvancalster reviewed (approved):
Ziet er goed uit. Wel nog een paar verwijzingen naar voorlopige protocols...
